corehq/apps/userreports/templates/userreports/reportbuilder/configure_report.html

```diff
@@ -92,7 +92,7 @@ <h3 data-bind="text: reportType() === 'agg' ?  '{% trans 'Indicators' %}' : '{%
               <td>
                 <select data-bind="fadeVisible: $parent.isAggregationEnabled,
                                    value: groupByOrAggregation">
-                  <option value="groupBy">{% trans 'Display Each' %}</option>
+                  <option value="groupBy">{% trans 'Group By' %}</option>
```

Comment:
Are you sure? iirc, original user testing suggested we not use "group by", and we picked "display each" instead. I could be wrong though.